geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloClientConfig.scala

```diff
@@ -20,14 +26,19 @@ import scala.util.Try
 /**
  * Client configuration options, loaded from a config file
  *
- * @param instance instance name
- * @param zookeepers zookeeper connect string
- * @param zkTimeout zookeeper timeout
+ * @param config Accumulo client configuration
+ * @param hasInstances instance id or name set in config file
```

Comment:
(Picky) Should this be "hasInstance"?

Reply:
Yeah I think so. I probably assumed that it was like the zookeeper setting where you could have multiple instances. Just pushed up a fix.